### PR TITLE
CSP-1428: Provider PR Outer API: Add Account Request

### PR DESCRIPTION
**Pull request by shaun-downey-education**
- Added new **RequestsController** with **AddRequest** method.

### CODE REVIEW

File: src/ProviderPR/SFA.DAS.ProviderPR/Application/Relationships/Queries/GetRelationships/GetRelationshipsQueryHandler.cs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿using MediatR;
 
-namespace SFA.DAS.ProviderPR.Application.Queries.GetRelationships;
+namespace SFA.DAS.ProviderPR.Application.Relationships.Queries.GetRelationships;
 public class GetRelationshipsQueryHandler : IRequestHandler<GetRelationshipsQuery, GetRelationshipsQueryResult>
```
Comment (MarkFCain) [inline src/ProviderPR/SFA.DAS.ProviderPR/Application/Relationships/Queries/GetRelationships/GetRelationshipsQueryHandler.cs:4]: sonarcloud reporting this needs coverage

File: src/ProviderPR/SFA.DAS.ProviderPR/Application/Requests/Commands/AddAccountRequestCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+        PostNotificationsCommand notificationCommand = new();
+
+        // If the provided EmployerContactEmail is null - then we must run through each account owner and send out the 'AddAccountOwnerInvitation' notification.
+
+        if (string.IsNullOrWhiteSpace(command.EmployerContactEmail))
+        {
+            CreateNotificationsForMembers(ref notificationCommand, command, teamMembers);
+
+            await SendNotifications(notificationCommand, cancellationToken);
```
Comment (MarkFCain) [inline src/ProviderPR/SFA.DAS.ProviderPR/Application/Requests/Commands/AddAccountRequestCommandHandler.cs:39]: remove comment and call this SendNotificationsToAddAccountOwnerInvitation(

Comment (shaun-downey-education) [reply src/ProviderPR/SFA.DAS.ProviderPR/Application/Requests/Commands/AddAccountRequestCommandHandler.cs:39]: @MarkFCain The send notifications method is called for all three Notification types and in some cases for multiple different types at the one time so has to be generic.

**Review by MarkFCain — APPROVED** [review]

**Comment by shaun-downey-education** [comment]
Parent pull request CSP-1468 incorporates both 1470 & 1428 as well as 1468.

**Pull request closed without merging**